### DISCUSSION TITLE
Unable to upgrade back to student account after getting verified

### DISCUSSION THREAD

**Discussion by lunzard** [community/community#35608] [GitHub Education]
Few weeks ago, I accidentally downgraded my verified student account to a free account. Now I want to upgrade it back to student account, but the system does not allow me to verify my student identity again,  as my previous verification was only 3 months ago. Does anyone know how I can fix the issue and upgrade my account?

**Comment by fadyat** [comment]
You are PRO already

**Reply by lunzard** [reply]
hi fadyat, thanks for the response!  Sadly, I was trying to purchase a pro account plan to see whether there is an option for me to convert from charged pro account to my original free student account. However, there is no option for me to convert it and I have to pay for this month's subscription.
